plugins/woocommerce/client/admin/client/marketplace/components/product-list-content/product-list-content.tsx

```diff
@@ -28,8 +28,14 @@ export default function ProductListContent( props: {
 } ): JSX.Element {
 	const wccomHelperSettings = getAdminSetting( 'wccomHelper', {} );
 
+	const rowsClassName =
+		props.products.length >= 8
```

Comment:
I doubt this is worth worrying about, but what if we want to show 3 rows? Is it better to futureproof this to some extent? AI suggestion:

```tsx
const ITEMS_PER_ROW = 4;
const rowCount = Math.ceil( props.products.length / ITEMS_PER_ROW );
const rowsClassName = `woocommerce-marketplace__product-list-content--${
	rowCount === 1 ? 'single-row' : `${ rowCount }-rows`
}`;
```